runtime/configs/types/keybase_ext.go

```diff
@@ -0,0 +1,25 @@
+package types
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+)
+
+func (k KeybaseType) MarshalJSON() ([]byte, error) {
+	return json.Marshal(KeybaseType_name[int32(k)])
```

Comment:
For my own understanding can you explain `KeybaseType_name` and `KeybaseType_value`? I am unsure what they are and where they come from.

Reply:
Sure, they are generated code for the enum type. This code exists so we can have the human-readable names in the config.json instead of ints.

```go
// keybase.pb.go

type KeybaseType int32

const (
	KeybaseType_FILE KeybaseType = 0
	// Hashicorp Vault
	KeybaseType_VAULT KeybaseType = 1
)

// Enum value maps for KeybaseType.
var (
	KeybaseType_name = map[int32]string{
		0: "FILE",
		1: "VAULT",
	}
	KeybaseType_value = map[string]int32{
		"FILE":  0,
		"VAULT": 1,
	}
)
```